bin/autodetect/CCmdLineParser.cc

```diff
@@ -62,6 +63,8 @@ bool CCmdLineParser::parse(int argc,
         desc.add_options()
             ("help", "Display this information and exit")
             ("version", "Display version information and exit")
+            ("config", boost::program_options::value<std::string>(),
+             "The configuration file")
```

Comment:
It's probably worth saying "The job configuration file", to hint that what's required is the job config